pilot/pkg/bootstrap/server.go

```diff
@@ -478,6 +473,7 @@ func (s *Server) initDiscoveryService(args *PilotArgs) error {
 	log.Infof("starting discovery service")
 	// Implement EnvoyXdsServer grace shutdown
 	s.addStartFunc(func(stop <-chan struct{}) error {
+		log.Infof("Starting ADS server")
```

Comment:
move log so its not triggered in tests. IMO its callers responsibility to log anyways